test/server/GameActions/SacrificeCard.spec.js

```diff
@@ -62,7 +62,10 @@ describe('SacrificeCard', function() {
             });
 
             it('moves the card to discard', function() {
-                expect(this.playerSpy.moveCard).toHaveBeenCalledWith(this.cardSpy, 'discard pile');
```

Comment:
Should remove `moveCard` method from the spy object at the top.

Reply:
good catch, fixed it and removed the playerSpy in the latest commit